### DISCUSSION TITLE
No WiFi Connection for Ender3 V3 KE

### DISCUSSION THREAD

**Discussion by RogersPhotography** [OrcaSlicer/OrcaSlicer#4063] [Q&A]
I just downloaded v 1.9 to try out and I followed all the installation instructions however when I try to connect WiFi to my Ender3 V3 KE I get a Error 404. It also says in a popup that I need Octoprint installed but I thought that was part of the Orca installation. Do I need a separate plugin to access my printer via network?

**Comment by liorabel** [comment]
Same issue here. Orca looks like a massively better slicer than Creality Print but as it is now it will add extra manual steps to a print that I'd rather avoid.

**Reply by RogersPhotography** [reply]
Yes, I've only been printing since December but quite prolifically. The KE is a very nice printer especially for the $ but the less than stellar Creality Print software and their non existent customer service has me eyeing Bambu or Prussia both of which have seemingly better slicers and customer service. I'm looking at the A1mini/AMS lite for my needs.
